actions.js

```diff
@@ -8,20 +8,26 @@ window.addEventListener("load", function () {
   let happinessStat = document.getElementById("happinesStat");
   let sleepStat = document.getElementById("sleepStat");
   let energyStat = document.getElementById("energyStat");
+  
 
   window.updateFrontendStatus = function () {
     hungerStat.textContent = pet.hunger;
     happinessStat.textContent = pet.happiness;
     sleepStat.textContent = pet.sleep;
     energyStat.textContent = pet.energy;
+  
+    if(hungerStat.textContent > 80){
```

Comment:
I'll prefere if you do this check from the `pet` "database" since there it will be a real integer instead of a string